Back-End/controllers/registration_controllers/login_controllers.js

```diff
@@ -1,5 +1,38 @@
-const db = require('../../db');
+const connection = require('../../db');
+require("dotenv").config();
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+
+const login = async (req,res)=>{
+    let email=req.body.email;
+    const query = await `SELECT * FROM users WHERE email ='${email}'`
+
+    connection.query(query,async(err,result)=>{
```

Comment:
To be consistent with the team code, do not use `async` and below use callbacks with ` bcrypt.compare`